lm_eval/api/utils.py

```diff
@@ -4,7 +4,7 @@
 import sys
 import numpy
```

Comment:
numpy doesn't seem to be used anymore

Reply:
Oh, gees. Somehow my editor didn't pick up on that. Thanks again :)